### PR TITLE
Use workflows in release-openshift-origin-installer-e2e-aws-upgrade

### PR DESCRIPTION
**Pull request by vrutkovs**
Port nightly upgrade job to workflows

### CODE REVIEW

**Comment by vrutkovs** [comment]
/hold

lets see if rehearsals would work

**Review by petr-muller — APPROVED** [review]

File: ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml [hunk 1]
```diff
@@ -1089,10 +1089,9 @@ periodics:
             openshift_installer:
               cluster_profile: "$(CLUSTER_TYPE)"
           - as: e2e-$(CLUSTER_TYPE)-upgrade
-            commands: TEST_SUITE=all run-upgrade-tests
-            openshift_installer:
-              cluster_profile: "$(CLUSTER_TYPE)"
-              upgrade: true
+          steps:
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:1092]: indentation looks off

/lgtm cancel

**Comment by petr-muller** [comment]
I don't think rehearsals will work here, they cannot sub variables in the `UNRESOLVED_CONFIG` content

**Comment by vrutkovs** [comment]
Right, that won't work - BRANCH cannot be substituted

**Pull request closed without merging**